client.go

```diff
@@ -51,30 +51,41 @@ const (
 )
 
 var (
-	reqLogTemplate = template.Must(template.New("request").Parse(`Sending request:
-Method: {{.Method}}
-URL: {{.URL}}
-Headers: {{.Headers}}
-Body: {{.Body}}`))
-
-	respLogTemplate = template.Must(template.New("response").Parse(`Received response:
-Status: {{.Status}}
-Headers: {{.Headers}}
-Body: {{.Body}}`))
+	reqLogTemplate = template.Must(template.New("request").Parse(`
+============================================================================================
```

Comment:
Optional: We can use `go:embed` here to avoid hardcoding the template in a string literal. See: https://pkg.go.dev/embed

Reply:
Done! Thanks for the tip 😃